jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/sftp/AutoConfiguration.java

```diff
@@ -39,9 +43,8 @@
 
     private SftpInputProperties properties;
 
-    public AutoConfiguration(SftpInputProperties sftpInputProperties) {
+    public AutoConfiguration(SftpInputProperties sftpInputProperties, ConcurrentMetadataStore redisMetadataStore) {
```

Comment:
I think you don't need it here, as it's a parameter of the bean method

Reply:
nice catch. I forgot to remove it.
done now.